src/media/css/base.styl

```diff
@@ -9,6 +9,7 @@ to this project.
 }
 #footzilla {
     background-image: url(../img/footer.png);
+    background-size: auto 22px;
```

Comment:
Thanks, you're on the right track here but IE and Safari have issues with `background-size: auto` so we should make this `background-size: 81px 21px` and have it apply to both regular and high pixel density screens.